packages/backend/src/plugins/search.ts

```diff
@@ -40,6 +40,10 @@ export default async function createPlugin(
 		factory: DefaultCatalogCollatorFactory.fromConfig(env.config, {
 			discovery: env.discovery,
 			tokenManager: env.tokenManager,
+			filter: {
+				// Filter out Location
+				kind: ['API', 'Component', 'Domain', 'Group', 'Resource', 'System', 'User'],
+			},
```

Comment:
The  [DefaultCatalogCollatorFactoryOptions has been deprecated](https://github.com/backstage/backstage/blob/b245b97e8c2a4baa0cb57bc2fed951e211cf46c4/plugins/search-backend-module-catalog/src/collators/DefaultCatalogCollatorFactory.ts#L48).
FYI:  It looks like the [filter is configurable through using a config key](https://github.com/backstage/backstage/tree/master/plugins/search-backend-module-catalog) in the app-config file instead. 


```
search:
  collators:
    catalog:
      filter:
        kind: ['API', 'Component', 'Domain', 'Group', 'Resource', 'System', 'User']
```

I'm fine making this change in code too, though. Just wanted to pass along incase you weren't aware

Reply:
Thanks for the heads up! I wasn't aware DefaultCatalogCollatorFactoryOptions was deprecated.
I switched it over to the app-config method.

Reply:
No problem! I haven't tested it out, but the changes look good based on their docs!